frontend/src/metabase/components/LastEditInfoLabel/LastEditInfoLabel.jsx

```diff
@@ -21,7 +21,7 @@ LastEditInfoLabel.propTypes = {
       id: PropTypes.number.isRequired,
       email: PropTypes.string.isRequired,
       first_name: PropTypes.string.isRequired,
-      last_name: PropTypes.string.isRequired,
+      last_name: PropTypes.string,
```

Comment:
This is from a change in 44 release https://github.com/metabase/metabase/pull/23154

Reply:
thanks for sharing, I'll remove `isRequired` for first_name as well in this case 😄